lib/api/apiUtils/object/setPartRanges.js

```diff
@@ -37,7 +37,7 @@ function setPartRanges(dataLocations, outerRange) {
     const max = end - begin + 1;
     let total = 0;
     for (let i = 0; i < dataLocations.length; i++) {
-        if (total >= max) {
+        if (total >= (max - 1)) {
```

Comment:
On line 36, it says we are adding 1 because range is inclusive. Why this change?